content/blog/2022/06/v1-release.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Kubewarden reaches 1.0.0 release 🎉
+authors:
+- Flavio Castelli
+date: 2022-06-22
+---
+
+Kubewarden is a policy engine for Kubernetes that is part of CNCF Sandobox.
```

Comment:
```suggestion
Kubewarden is a policy engine for Kubernetes that is part of CNCF Sandbox.
```

Reply:
I would go so far as to change it and break it into two separate sentences as below:
```suggestion
Kubewarden is a policy engine for Kubernetes. The project is a part of [CNCF Sandbox](https://www.kubewarden.io/blog/2022/06/cncf-sandbox-inclusion/).
```